External/FEXCore/include/FEXCore/IR/IR.h

```diff
@@ -287,29 +287,29 @@ struct MemOffsetType final {
 };
 
 struct TypeDefinition final {
-  uint8_t Val;
-  operator uint8_t() const {
+  uint16_t Val;
```

Comment:
Was this size change just to support vector element counts up to 255 elements in size?

Reply:
Yes. The old setup supported vectors up to 3 elements, which is not enough, and packed it on 8 bits.